content/SCALE/SCALETutorials/Apps/CommunityApps/MinIOApp/MinIOClustering.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Setting Up MinIO Clustering"
+description: "Provides configuration instructions using the MinIO Offical Charts application widget. It includes instructions on setting up a distributed cluster configuration."
+weight: 20
+aliases:
+ - /scale/scaleuireference/apps/minioclustersscale/
+ - /scale/scaletutorials/apps/minioclustersscale/minioclustering/
+tags:
+- scaleminio
+- scaleapps
+- scaleclustering
+---
+
+{{< toc >}}
+
+
+{{< hint info >}}
+This article applies to the public release of the S3 **MinIO** charts application in the TRUENAS catalog.
+{{< /hint >}}
+
+On TrueNAS SCALE 23.102-ALPHA.2 and later, users can create a MinIO S3 distributed instance to scale out and handle individual node failures. 
+A node is a single TrueNAS storage system in a cluster.
+
+The examples below use four TrueNAS systems to create a distributed cluster.
+For more information on MinIO distributed setups, refer to the [MinIO documentation](https://docs.min.io/docs/distributed-minio-quickstart-guide.html).
+
+## First Steps
+
+{{< include file="/_includes/MinIODatasetRequirements.md" type="page" >}}
+
+For a distributed configuration, repeat this on all system nodes in advance. 
+
+Take note of the system (node) IP addresses or host names and have them ready for configuration. Also, have your S3 user name and password ready for later.
+
+## Configuring MinIO
+
+Configure the MinIO application using the full version **Minio charts** widget. 
```

Comment:
Does it say **Minio charts** in the UI? Or should this be something like **MinIO** widget from the **charts** train. Or, even more simply, community **MinIO** widget?

Reply:
The widget says the train is charts, but I get your point with this. I shall reword this.